### ISSUE TITLE
[refineDet] i did the retrain based on the test.py, but the loss is aroud 4.9. and the tp is 0.....

### ISSUE THREAD

**Issue by sycophant-stone** [Stick-To/Object-Detection-API-Tensorflow#18]
could anybody share your pretrained weight?
thanks alot
